doc/source/index.rst

```diff
@@ -1,116 +1,108 @@
-SciPy
-=====
+.. template taken from Pandas
 
-:Release: |release|
-:Date: |today|
+.. module:: scipy
 
-SciPy (pronounced "Sigh Pie") is open-source software for mathematics,
-science, and engineering.
+*******************
+SciPy documentation
+*******************
 
-.. toctree::
-   :maxdepth: 1
+**Date**: |today| **Version**: |version|
 
-   install_upgrade
-   api
-   release
+**Download documentation**: `PDF Version <https://docs.scipy.org/doc/scipy-1.6.1/scipy-ref-1.6.1.pdf>`__ | `Zipped HTML <https://docs.scipy.org/doc/scipy-1.6.1/scipy-html-1.6.1.zip>`__
 
-Tutorial
---------
+**Useful links**:
+`Binary Installers <https://pypi.org/project/scipy>`__ |
+`Source Repository <https://github.com/scipy/scipy>`__ |
+`Issues & Ideas <https://github.com/scipy/scipy/issues>`__ |
+`Q&A Support <https://stackoverflow.com/questions/tagged/scipy>`__ |
+`Mailing List <https://mail.python.org/pipermail/scipy-user>`__
 
-Tutorials with worked examples and background information for most SciPy
-submodules.
+:mod:`scipy` (pronounced "Sigh Pie") is open-source software for mathematics,
```

Comment:
This link appears to not be working, because it refers to this page. So better leave it out and write as `SciPy`.